### PR TITLE
27 basic styling

### PR DESCRIPTION
**Pull request by ljreaux**
This is the bare bones styling based on the style doc. In order to look how I intended several things need to be done that will cause merge conflicts for ya'll so I didn't include them in these commits, so you should on the components you are working on.

1. All images in the entire project need to have their sizing attributes removed from the actual element. Image sizing is now handled by the CSS.
2. The userUpdateForm  on the account page REALLY needs to be inside of the userInfo div.
3. SingleRecipe instructions need to be in their own div for me to proceed any further. Also comments and Reviews need their own divs on the page.
4. The form inside of the NewRecipe component needs to have a className of newRecipeForm.
5. Consider moving the input elements outside of the labels and linking via id on the login page. This will allow for easier alignment of elements. 

Things that still need to be worked on once completed.
1. The rest of the single recipe page.
2. styling the searchbar.
3. adding binder rings to single recipe. https://codepen.io/ljreaux/pen/rNbaOyM
4. changing navs on the account and admin pages to look like folder tabs. https://css-tricks.com/better-tabs-with-round-out-borders/

Additional styling will need to be completed as other components are finished.

### CODE REVIEW

**Review by jared23m — APPROVED** [review]

**Review by BarryFullStack — APPROVED** [review]
I think we should swap around some of the colors. But I like how this is handled. I can make up some mock up color suggestions and put them in slack

File: FE/src/index.css [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+  overflow-x: hidden;
+  padding: 0;
+  margin: 0;
+  background-color: var(--clr-accent-four);
+}
+#root {
+  font-family: "Fira Sans Condensed", sans-serif;
+  font-weight: 400;
+  color: var(--clr-accent-two);
+  background-color: var(--clr-accent-four);
```
Comment (BarryFullStack) [inline FE/src/index.css:22]: background-color: var(--clr-primary);

File: FE/src/index.css [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+  --clr-accent-two: #5b5731;
+  --clr-accent-three: #928d51;
+  --clr-accent-four: #ebe9e8;
+}
+body {
+  width: 100vw;
+  overflow-x: hidden;
+  padding: 0;
+  margin: 0;
+  background-color: var(--clr-accent-four);
```
Comment (BarryFullStack) [inline FE/src/index.css:16]: background-color: var(--clr-primary);

File: FE/src/index.css [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+  border-radius: 0.5em;
+  cursor: pointer;
+}
+
+button:hover,
+button:focus,
+input[type="button"]:hover,
+input[type="button"]:focus {
+  background-color: var(--clr-accent-three);
```
Comment (BarryFullStack) [inline FE/src/index.css:54]: color: var(--clr-accent-four);

File: FE/src/index.css [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+
+img {
+  max-width: 100%;
+}
+
+button,
+input[type="button"] {
+  font-family: "Fira Sans Condensed", sans-serif;
+  font-weight: 400;
+  color: var(--clr-primary);
```
Comment (BarryFullStack) [inline FE/src/index.css:43]: color: var(--clr-secondary);

File: FE/src/index.css [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+  gap: 0;
+}
+
+.recipesContainer {
+  display: grid;
+  place-items: center;
+}
+
+.recipeCard {
+  border: 1px solid var(--clr-accent-two);
```
Comment (BarryFullStack) [inline FE/src/index.css:134]: background-color: var(--clr-accent-four);

**Review by amead99 — APPROVED** [review]

**Pull request merged**